dev/ush/gw_setup.sh

```diff
@@ -8,8 +8,16 @@
 # This script should be SOURCED to properly setup the environment.
 #
 
-HOMEgfs="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
+HOMEgfs="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd)"
 source "${HOMEgfs}/ush/detect_machine.sh"
 source "${HOMEgfs}/ush/module-setup.sh"
 module use "${HOMEgfs}/modulefiles"
 module load "module_gwsetup.${MACHINE_ID}"
+
+# Set up the PYTHONPATH for wxflow if it is not already loaded
+if ! python3 -c "import wxflow" 2>/dev/null; then
+  if [[ -d "${HOMEgfs}/sorc/wxflow/src" ]]; then
+    PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${HOMEgfs}/sorc/wxflow/src"
```

Comment:
Yay, finally we have `PYTHONPATH `set uniformly for **wxflow** in one place.
Is there a timeline for getting **wxflow** in Stack Spack via PyPI so we can remove this too?

Reply:
I think ss-1.9.x has wxflow

Reply:
Yes, it does: `/autofs/ncrc-svm1_proj/epic/spack-stack/c6/spack-stack-1.9.2/envs/ue-intel-2023.2.0/install/modulefiles/intel/2023.2.0/py-wxflow/0.2.0`.